power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/services/MoneyTransferService.java

```diff
@@ -0,0 +1,41 @@
+package com.adorsys.gis.powerpay.powerpaybackend.services;
+
+import com.adorsys.gis.powerpay.powerpaybackend.domain.Transaction;
+import com.adorsys.gis.powerpay.powerpaybackend.repository.MoneyTransferRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MoneyTransferService {
```

Comment:
De we have an interface for this already?

Reply:
Maybe this one?
power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/services/MoneyTranserInterface.java

Reply:
If yes, then why not using it?

Reply:
Yes @stephane-segning we do.

Reply:
We should use it then

Reply:
I'm working on it now sir